resources/shared-db/Cargo.toml

```diff
@@ -12,6 +12,7 @@ mongodb = { version = "2.3.0", optional = true }
 serde = { version = "1.0.148", features = ["derive"] }
 shuttle-service = { path = "../../service", version = "0.29.0" }
 sqlx = { version = "0.7.1", optional = true }
+tokio = { version = "1", features = ["time"] }
```

Comment:
I've added tokio. but we might not want to force users to use tokio. I'm not sure how to handle that otherwise.

Reply:
Tokio is already forced from the runtime.